__tests__/util/request-manager.js

```diff
@@ -34,3 +34,38 @@ test('RequestManager.request with cafile', async () => {
   }
   expect(body).toBe('ok');
 });
+
+test('RequestManager.execute Request 403 error', async () => {
+  const config = await createConfig({});
+  jest.mock('request', (factory) => (options) => {
+    options.callback('', {statusCode: 403}, '');
+    return {
+      on: () => {},
+    };
+  });
+  await config.requestManager.execute({
+    params: {url: `https://localhost:port/?nocache`, headers: {Connection: 'close'}},
+    resolve: (body) => {},
+    reject: (err) => {
+      expect(err.message).toBe('https://localhost:port/?nocache: Request "https://localhost:port/?nocache" returned a 403');
```

Comment:
It might be possible to wrap the `execute` call in a `Promise`, then call `reject` on the err, then `await` the `Promise` and `catch` the error and then follow it up with the `expect`. I'm not positive that would work, but I don't believe the `execute` returns a `Promise` so calling `await` on it shouldn't have the desired effect.